m/mongo-go-driver/mongo-go-driver_ubi_8.5.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : mongo-go-driver
+# Version       : v1.1.3
+# Source repo   : https://github.com/mongodb/mongo-go-driver
+# Tested on     : UBI: 8.5
+# Language      : Go
+# Travis-Check  : False
```

Comment:
@shreyakajbaje please provide additional comments on why the Travis flag is set as False